tests/braynsTestData.cpp

```diff
@@ -222,6 +222,31 @@ BOOST_AUTO_TEST_CASE(render_circuit_with_particle_renderer)
                                  brayns.getEngine().getFrameBuffer()));
 }
 
+BOOST_AUTO_TEST_CASE(render_demo_with_proximity_renderer)
+{
+    auto& testSuite = boost::unit_test::framework::master_test_suite();
+
+    const char* app = testSuite.argv[0];
+    const char* argv[] = {app,         "--renderer",
+                          "proximity", "--samples-per-pixel",
+                          "16",        "--synchronous-mode",
+                          "true",      "demo"};
+    const int argc = sizeof(argv) / sizeof(char*);
+
+    brayns::Brayns brayns(argc, argv);
+
+    const brayns::Vector3f rotCenter = {0.5f, 0.5f, 0.5f};
+
+    auto& camera = brayns.getEngine().getCamera();
+    const auto camPos = camera.getPosition();
+
+    camera.setTarget(rotCenter);
+    camera.setPosition(camPos - (rotCenter - camPos));
+
+    brayns.render();
```

Comment:
After my previous PR, you have to call commitAndRender() here now to fix the test error in CI.

Reply:
Thanks for the tip.

Reply:
hum... fails in debug. 
`Test is aborted
Leaving test module "braynsAddModelFromBlob"; testing time: 2179609us
LeakSanitizer: bad pointer 0x000000000056`